lua/starfall/libs_cl/render.lua

```diff
@@ -868,6 +887,22 @@ function render_library.setRGBA(r, g, b, a)
 	surface.SetTextColor(r, g, b, a)
 end
 
+--- Gets the drawing tint.
+-- @return Color Color (in 0-255 range)
+function render_library.getTint()
+	local r, g, b = render.GetColorModulation()
+	local a = render.GetBlend()
+		
+	return Color(r * 255, g * 255, b * 255, a * 255)
+end
+
+--- Sets the drawing tint.
+-- @param Color c Color (in 0-255 range)
+function render_library.setTint(c)
+	render.SetColorModulation(c.r / 255, c.g / 255, c.b / 255)
```

Comment:
I think you can use numerical indices `1-4` to avoid invoking the `__index` metaevent of a color object.